app/cinema/bar.py

```diff
@@ -0,0 +1,5 @@
+class CinemaBar:
+
+    @staticmethod
+    def sell_product(product: str, customer: str) -> None:
```

Comment:
"customer" type should be Customer type
```suggestion
    def sell_product(product: str, customer: Customer) -> None:
```